components/TextFields/src/MDCTextInputController.h

```diff
@@ -82,7 +82,7 @@
 /**
  Default value for disabledColor.
 
- Default is blue.
+ Default is [UIColor lightGrayColor].
```

Comment:
Super-duper-nit: `lightGrayColor` is a property. Prefer either

    Default is .lightGrayColor.

or

    Default is UIColor.lightGrayColor.

Reply:
oh yeah! But what you didn't notice is that all these `Default is [something real]` are false! This is a protocol. I have now updated it to delete them all. (There used to be so few controllers that they weren't a lie. They are lies now.)